### PR TITLE
New service: AstroBotList

### PR DESCRIPTION
**Pull request by EndBug**
Ref issue #58

### CODE REVIEW

**Review by Snazzah — CHANGES_REQUESTED** [review]
Should add a `getOwnStats` function since it seems to be an endpoint [here](https://github.com/sell/abl-wrapper/blob/master/src/index.js#L51)

File: typings/index.d.ts [hunk 1]
```diff
@@ -510,15 +533,9 @@ export class DiscordBotsCo extends ServiceBase {
...
- * @param options - The options of the service. Providing this is highly recommended.
- * @param options.library - The bot's library
- * @param options.clientID - The bot ID for the user agent
  */
 export class DiscordBotsGG extends ServiceBase {
-    constructor(token: string, options: {
-        library: string;
-        clientID: IDResolvable;
-    });
+    constructor(token: string);
```
Comment (Snazzah) [inline typings/index.d.ts:538]: Why did this change?

Comment (EndBug) [reply typings/index.d.ts:538]: I have no idea of what happened, I'll run the generator again to see what happens

**Comment by EndBug** [comment]
Superseded by #98

**Pull request closed without merging**